tests/virt_autotest/switch_to_ssh_and_install_hypervisor.pm

```diff
@@ -19,25 +19,25 @@ use base 'virt_autotest_base';
 use testapi;
 use ipmi_backend_utils;
 use Utils::Backends qw(use_ssh_serial_console is_remote_backend);
-use utils qw(zypper_call systemctl permit_root_ssh_in_sol);
+use utils qw(zypper_call systemctl permit_root_ssh_in_sol script_retry);
 use virt_autotest::utils qw(is_kvm_host is_xen_host);
 
 sub run {
 
     #enable ssh root access for openSUSE TW in sol console
     #root access is not permitted by default in TW, see bsc#1173067#c2
     select_console 'sol', await_console => 0;
+    save_screenshot;
     for (my $i = 0; $i <= 4; $i++) {
         send_key 'ret' if check_screen('sol-console-wait-typing-ret', 60);
         last if (check_screen([qw(linux-login text-login)], 60));
         save_screenshot;
     }
 
     if (check_screen('text-login')) {
-        enter_cmd "root";
-        assert_screen "password-prompt";
-        type_password;
-        send_key('ret');
+        sol_login();
+        # Try one more time as TW login fails with incorrect password sporatically in O3
+        sol_login() if check_screen('sol-login-password-incorrect', 60);
```

Comment:
I personally do not prefer a new needle like this on sol console, especially an "incorrect one". I think 
```if !check_screen("logged in as root in sole")  retry sol_login```
is better.

Reply:
The test still experiencing password typing problem. I leave the commit in next PR, only merge other commits in the PR.